### ISSUE TITLE
bad results with qwen edit-2511

### ISSUE THREAD

**Issue by BenDes21** [kohya-ss/musubi-tuner#840]
Hi there, is anyone getting bad result with qwen edit-2511 compare to previous version with same dataset and same settings ( except --model_version and files )

Here my parameters :   

accelerate launch --num_cpu_threads_per_process 1 --mixed_precision fp16 src/musubi_tuner/qwen_image_train_network.py \
--max_train_epochs 200 --save_every_n_epochs 10 --seed 42 \
--output_dir /workspace/output --output_name test_V3 \
--dataset_config /workspace/dataset.toml \
--dit /workspace/models/qwen_image_edit_2511_bf16.safetensors --model_version edit-2511 \
--text_encoder /workspace/models/qwen_2.5_vl_7b.safetensors \
--vae /workspace/models/qwen_image_vae.safetensors \
--mixed_precision bf16 --fp8_base --fp8_scaled --gradient_checkpointing --sdpa  \
--timestep_sampling shift --weighting_scheme none --discrete_flow_shift 2.2 \
--max_data_loader_n_workers 12 --persistent_data_loader_workers \
--network_dim 16 --network_module networks.lora_qwen_image \
--optimizer_type schedulefree.RAdamScheduleFree --learning_rate 2e-5

dataset 

[general]
resolution = [1024, 1024] 
caption_extension = ".txt" 
enable_bucket = true 
bucket_no_upscale = false 

[[datasets]]
image_directory = "/workspace/test/dataset"
cache_directory = "/workspace/test/dataset" 
control_directory = "/workspace/test/control"
qwen_image_edit_no_resize_control = false 
qwen_image_edit_control_resolution = [1024, 1024]

**Comment by Superbelko** [comment]
It is hard to tell what kind of bad results you experience, but for me it seems the editing results is messed up without using lightning lora with qwen_edit_2511_bf16 model, i tried training for 6000 steps a 20 image pairs dataset and it is definitely looks overfit.
No idea if it is same for 2509 since I don't have it right now.
Was hoping for it to stabilize but it still looks messed up so i am forced to use lightning lora.
I think it could somehow messing up with the model because I do not use activation tags or special unique keywords but do not have time to test if that is reason.

My lora is focusing on character and the character becomes classic early SD body horror when using 2x resolution completely messed up with no head and body or simply skin blobs, with the background mostly intact, works as expected with 4 steps lightning lora though.
